.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
To chyba nie jest plik projektu? :)

Reply:
Konfiguracja do uruchamiania debuggera. Można się kłócić że przydatna, ale fakt, że nie powinna się tu znaleźć. :-D

Reply:
Usunąłem